ragged/src/chat/Chat.test.ts

```diff
@@ -633,6 +633,11 @@ describe("Chat", () => {
         expect(c).toBeDefined();
       })
 
+      it('can instantiate ollama', () => {
+        const c = Chat.with('ollama', { apiKey: '123' });
```

Comment:
<img width="1365" alt="Screenshot 2024-07-02 at 09 39 00" src="https://github.com/monarchwadia/ragged/assets/5929807/3d8b48fa-4836-4c05-aacd-4ed8712faecb">

This fails with a type error. Not sure why the others work but this doesn't. What have I done wrong? 😕